.github/workflows/make_changelogs.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   MakeCL:
     runs-on: ubuntu-latest
-    if: github.repository == 'BeeStation/BeeStation-Hornet'
+    if: github.repository == 'BeeStation/BeeStation-Hornet' && !contains(github.event.head_commit.message, '[ci skip]')
```

Comment:
Modify to our repository.

Reply:
There are a few of these that I need to correct actually. I have to go through all the github files.